### DISCUSSION TITLE
IOS系统问题

### DISCUSSION THREAD

**Discussion by YAHAHA02** [getlantern/lantern-forum-cn#648] [Q&A]
IOS系统链接不上 也没有专业版同步 有没有教程请指点一下@Mike777Khoo
![d5010dae70d56a0a9f7420a6326e946](https://github.com/getlantern/lantern-forum-cn/assets/165741830/8cd77497-9174-4f3d-98dd-a6598904a6af)
![e5f455e4e5b92a1c52e20c3206aaf64](https://github.com/getlantern/lantern-forum-cn/assets/165741830/df7eba6f-5dc3-48ed-8bd3-309c6896d03d)

**Comment by MaxMKMax** [comment]
@YAHAHA02 
您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。
